http/1.7/modules/ROOT/pages/http-polling-source-ref.adoc

```diff
@@ -0,0 +1,306 @@
+= HTTP Request Configuration Reference
+:page-aliases: connectors::http/http-polling-source-ref.adoc
+
+The HTTP Polling Source allows you to have an event source that, instead of waiting for an HTTP request from a user (like the HTTP Listener), performs an scheduled HTTP Request to a given HTTP Server.
+
+You can configure the scheduling frequency. May it be a fixed frequency, or a cron expression. You can also configure how to split the HTTP Response from the server, as well how to retrieve watermarking (if any) and ids (if any).
+
+== Relationship with HTTP Request Operation
+
+Since this Event Source performs an HTTP Request, as the operation does, this event source uses an HTTP Request Configuration as well. There, you can set the url, port, basepath, protocol, as well as any other request configuration needed. 
+
+This Polling Source will execute an HTTP Request, similar to the one performed in the operation, and with the resulting response it will create an event (or more than one) to execute its flow. 
+
+So, for example: 
+
+[source,xml,linenums]
+----
+<http:request-config name="requestConfig">
+    <http:request-connection host="localhost" port="8081"/>
+</http:request-config>
+
+<flow name="example-polling-source">
+    <http:polling-source config-ref="requestConfig" path="/test">
+        <scheduling-strategy >
+            <fixed-frequency frequency="10" timeUnit="HOURS"/>
+         </scheduling-strategy>        
+    </http:polling-source>
+    <logger message="#[payload]"/>
+</flow>
+----
+
+== Configuring Polling Frequency
+
+In the previous example, we had a Polling Source that polled every 10 hours. We can configure this by changing the scheduling strategy in the Event Source element:
```

Comment:
This is the same for every polling source

Reply:
It is, but shouldn't we show it either way? I leave that to the docs team so it's consistent to everything else